modules/light-clients/08-wasm/types/client_state_test.go

```diff
@@ -213,23 +213,35 @@ func (suite *TypesTestSuite) TestValidate() {
 }
 
 func (suite *TypesTestSuite) TestInitializeGrandpa() {
-	var consensusState exported.ConsensusState
+	var (
+		consensusState exported.ConsensusState
+		clientState    exported.ClientState
+	)
 	testCases := []struct {
```

Comment:
wouldn't say no to a cheeky line-break here 👀 

```suggestion
	)
	
	testCases := []struct {
```

Reply:
up for cheeky line-breaks any day